### PR TITLE
Add support for component updater

### PR DESCRIPTION
**Pull request by bbondy**
Auditors: @bridiver @darkdh

### CODE REVIEW

**Comment by darkdh** [comment]
Could we pull out an `script/lib/update_client.py` just like [script/lib/autofill.py](https://github.com/brave/libchromiumcontent/blob/master/script/lib/autofill.py) does ?

**Comment by bbondy** [comment]
yes good idea, I'll do that tomorrow.

File: script/create-dist [hunk 1]
```diff
@@ -60,41 +60,57 @@ BINARIES = {
   'darwin': [
     'chromedriver',
     'mksnapshot',
+    'libupdate_client.a',
```
Comment (bridiver) [inline script/create-dist]: can you also move the dist files to an update_client.py file?

Comment (bbondy) [reply script/create-dist]: this is updated now

**Comment by bridiver** [comment]
not sure why this didn't auto-close, but merged manually

**Pull request closed without merging**